src/cameras/CubeCamera.js

```diff
@@ -58,6 +58,8 @@ function CubeCamera( near, far, cubeResolution, options ) {
 
 		if ( this.parent === null ) this.updateMatrixWorld();
 
+		var prevRenderTarget = renderer.getRenderTarget();
```

Comment:
This pattern is used elsewhere in the library, and the variable name is `currentRenderTarget`. For consistency, I suggest using the same name.

Reply:
Makes sense. Done.